app/graphql/types/objects/base/attachment_type.rb

```diff
@@ -21,13 +23,22 @@ def filename
       end
     end
 
-    def url
-      if object.class.eql?(ActiveStorage::Variant)
-        Rails.application.routes.url_helpers.rails_representation_url(object)
-      elsif defined?(object.service_url)
-        object.service_url
+    def url(resolution:)
```

Comment:
@anbublacky - should we check here if the attachment is an image and only then proceed to try and resize?

Reply:
This is resolved